core/src/main/java/org/apache/brooklyn/core/entity/EntityPredicates.java

```diff
@@ -454,12 +454,6 @@ public String toString() {
         }
     }
 
-    /** @deprecated since 0.7.0 use {@link #locationsIncludes(Location)} */
-    @Deprecated 
-    public static <T> Predicate<Entity> withLocation(final Location location) {
-        return locationsIncludes(location);
-    }
-    
     /** @deprecated since 0.7.0 use {@link #locationsIncludes(Location)}, introduced to allow deserialization of anonymous inner class */
     @SuppressWarnings("unused") @Deprecated 
     private static <T> Predicate<Entity> withLocationOld(final Location location) {
```

Comment:
Can we remove this one? Don't think it is use for rebind

Reply:
If someone has really old historic persisted state then they might reference this anonymous class!

At some point (when we have a way for the user to fix their persisted state easily?!) we'll delete this stuff.